sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -2167,6 +2167,14 @@ object SQLConf {
     .booleanConf
     .createWithDefault(false)
 
+  val DEDUPLICATE_MAP_KEY_WITH_LAST_WINS_POLICY =
+    buildConf("spark.sql.deduplicateMapKey.lastWinsPolicy.enabled")
+      .doc("When true, use last wins policy to remove duplicated map keys in built-in functions, " +
+        "this config takes effect in below build-in functions: CreateMap, MapFromArrays, " +
+        "MapFromEntries, StringToMap, MapConcat and TransformKeys.")
```

Comment:
We should mention if false then an exception will be thrown.

Reply:
Thanks, done in fe1fb0f.